packages/react/src/side-nav/SideNav.tsx

```diff
@@ -18,21 +18,24 @@ export type SideNavProps = {
 	activePath: string;
 	/** Used as the title of the expand/collapse trigger on smaller screen sizes. */
 	collapseTitle: string;
-	/** If SideNav is placed on 'bodyAlt' background, please set this to 'bodyAlt'. */
-	background?: CollapsingSideBarBackground;
 	/** The list of links. */
 	items: SideNavMenuItemType[];
 	/** The title is placed at the top of the list of links. */
 	title: string;
+	/** If SideNav is placed on 'bodyAlt' background, please set this to 'bodyAlt'. */
+	background?: CollapsingSideBarBackground;
+	/** The variant to use for nested navigation items. */
+	nestedItemsVariant?: 'always-open' | 'open-on-nav';
 	/** If provided, the title will be rendered as an anchor element. */
 	titleLink?: string;
 };
 
 export function SideNav({
 	activePath,
+	background = 'body',
 	collapseTitle,
 	items,
-	background = 'body',
+	nestedItemsVariant = 'open-on-nav',
```

Comment:
This _may_ be a breaking change for websites with a small number navigation items utilising nesting. We would want them to add `nestedItemsVariant="always-open"`. May need to revert this to current default (only) behaviour?

Reply:
Yeah, it feels like it would be a breaking change

Reply:
Ok, I'll revert and share.

- [ ] revert default
- [ ] share with team